src/Numerics/Mesh/Topologies.jl

```diff
@@ -2024,27 +2024,12 @@ function cubedshelltopowarp(
     b,
     c,
     R = max(abs(a), abs(b), abs(c));
-    r_inner = _planet_radius,
-    r_outer = _planet_radius + domain_height,
+    r_inner = 0,
+    r_outer = 1,
     topography::AnalyticalTopography = NoTopography(),
 )
+    f = get_topography_function(topography, r_inner, r_outer)
```

Comment:
@charleskawczynski here's how I handled the code duplication

Reply:
Why couldn't we call the existing function? Wasn't it exactly the same?

Reply:
it relied on planet_radius and domain_height and explicitly called analytical topography... which is unnecessary if you don't have topography. also causes the code to crash if planet_radius and domain_height are undefined, which is pretty common in the numerics tests. 

but also see this comment by @valeriabarra: https://github.com/CliMA/ClimateMachine.jl/pull/2171#discussion_r611946215

Reply:
dropped this change based on @valeriabarra's comment